website/server/feed.js

```diff
@@ -1,94 +0,0 @@
-/* eslint-disable sort-keys */
```

Comment:
Aren't we providing RRS feed anymore?

Reply:
Docusaurus doesn't have an RSS feature. I've made it a TODO.

Reply:
New PR includes updated Docusaurus with RSS/Atom feeds.